src/runtime/runtime.ts

```diff
@@ -35,16 +36,19 @@ export type RuntimeArcOptions = Readonly<{
   inspectorFactory?: ArcInspectorFactory;
 }>;
 
+let runtime: Runtime = null;
+
 // To start with, this class will simply hide the runtime classes that are
 // currently imported by ArcsLib.js. Once that refactoring is done, we can
 // think about what the api should actually look like.
 export class Runtime {
   private cacheService: RuntimeCacheService;
   private loader: Loader | null;
-  private composerClass: new () => SlotComposer | null;
+  private composerClass: typeof SlotComposer | null;
   public readonly context: Manifest;
   private readonly ramDiskMemory: VolatileMemory;
   readonly arcById = new Map<string, Arc>();
+  pecFactory;
```

Comment:
I think we can add the PecFactory type to this: 
`public pecFactory: PecFactory;`.

Reply:
Thanks

Reply:
Turns out it's not so easy, the `PecFactory` type has diverged from what `pecIndustry` is producing. It should be fixed, but I think it's out of scope for this PR.

I'll add it to the list for the larger Runtime update project.